subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -67,235 +67,134 @@
-                        </ul>
-
-                        <h3 id="integrating-gradle">Integrating Gradle</h3>
-                        <ul>
-                            <li><a class="nav-dropdown" data-toggle="collapse" href="#ci-integration" aria-expanded="false" aria-controls="ci-integration">Continuous Integration</a>
-                                <ul id="ci-integration">
-                                    <li><a href="https://guides.gradle.org/executing-gradle-builds-on-jenkins">Using Gradle with Jenkins</a></li>
-                                    <li><a href="https://guides.gradle.org/executing-gradle-builds-on-teamcity">Using Gradle with TeamCity</a></li>
-                                    <li><a href="https://guides.gradle.org/executing-gradle-builds-on-travisci">Using Gradle with Travis CI</a></li>
+                            <li><a class="nav-dropdown" data-toggle="collapse" href="#best-practices" aria-expanded="false" aria-controls="best-practices">Best practices</a>
+                                <ul id="best-practices">
+                                    <li><a href="/userguide/authoring_maintainable_build_scripts.html">Authoring maintainable build scripts</a></li>
+                                    <li><a href="/userguide/organizing_gradle_projects.html">Organizing Gradle projects</a></li>
+                                    <li><a href="https://guides.gradle.org/performance/">Optimizing build performance</a></li>
+                                    <li><a href="https://guides.gradle.org/using-build-cache/">Using the build cache</a></li>
                                 </ul>
                             </li>
-                            <li><a class="nav-dropdown" data-toggle="collapse" href="#ide-integration" aria-expanded="false" aria-controls="ide-integration">IDE Integration</a>
-                                <ul id="ide-integration">
-                                    <li><a href="/userguide/eclipse_plugin.html">Eclipse Plugin</a></li>
-                                    <li><a href="/userguide/idea_plugin.html">IDEA Plugin</a></li>
+                            <li><a class="nav-dropdown" data-toggle="collapse" href="#sample-gradle-builds" aria-expanded="false" aria-controls="sample-gradle-builds">Sample Gradle builds</a>
+                                <ul id="sample-gradle-builds">
+                                    <li><a href="https://github.com/gradle/gradle/tree/master/subprojects/docs/src/samples">Groovy DSL samples</a></li>
+                                    <li><a href="https://github.com/gradle/kotlin-dsl/tree/master/samples">Kotlin DSL samples</a></li>
                                 </ul>
                             </li>
-                            <li><a href="/userguide/embedding.html">Tooling API</a></li>
+                            <li><p></p></li>
+                            <li><a href="/userguide/feature_lifecycle.html">Gradle feature lifecycle</a></li>
```

Comment:
This feels a bit orphaned from the group, as if it should be bolded like "third party integration". Does it deserve a spot in the navigation at all? We link to it every time we say "incubating"

Reply:
I would honestly prefer not to have it in the navigation. It's not something you need to understand in order to work on a build. It's more aligned with something like the glossary. Perhaps we'll have it as a link at the end of the nav alongside _Glossary_ when that's done.

In the meantime, I think we could remove the link and link the page from the upgrade guides.

Reply:
- [x] Remove the link to Gradle feature lifecycle from the user manual nav
 - [ ] Link to that page from the upgrade guides